api/src/test/java/org/openmrs/module/ohricore/impl/HIVStatusComputedConceptTest.java

```diff
@@ -0,0 +1,106 @@
+package org.openmrs.module.ohricore.impl;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.ohricore.api.impl.HIVStatusComputedConcept;
+import org.openmrs.module.ohricore.engine.CommonsUUID;
+import org.openmrs.module.ohricore.engine.HIVStatusConceptUUID;
+import org.openmrs.test.jupiter.BaseContextSensitiveTest;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+/**
+ * @author MayanjaXL, Amos, Stephen, smallGod date: 07/07/2021
+ */
+public class HIVStatusComputedConceptTest extends BaseContextSensitiveTest {
+	
+	protected static final String OBS_ATLEAST_ONE_POSITIVE_DATA_XML = "org/openmrs/module/ohricore/include/HIVComputedConcept-AtleastOnePositiveObs.xml";
```

Comment:
@smallgod Move all the sample data into a single file, call it OHRI-StandardDataset.xml 

You can build off patients in this data set which is automatically loaded by the test harness 

https://github.com/openmrs/openmrs-core/blob/f76ee93cc46271997e577ed7fa223ac4a4e53dcb/api/src/test/resources/org/openmrs/include/standardTestDataset.xml

Reply:
Okay @ssmusoke thanks, let me do that. I will ping you later on.

Reply:
@ssmusoke went through one by one files not to track. I think all unnecessary files are now not being tracked. This is done.

Reply:
@smallgod You should delete the .idea folder then commit that change which IMO is the only way to remove the files from this version control

Reply:
@ssmusoke Yes, I deleted the .idea folder from my project and since it was not being tracked anymore. Git doesn't show any changes to commit (nothing to commit). And also shortly after deleting it, it re-created itself. Looks like it the IDE needs it.